package.json

```diff
@@ -15,7 +15,20 @@
     "biojs"
   ],
   "main": "build/pileup.js",
-  "browser": "build/pileup.js",
+  "browser": "src/pileup.js",
+  "browserify": {
```

Comment:
Does BioJS run `browserify` on its own?

If we need to support that, can you either:

1. Add a note to the corresponding sections of `Gruntfile.js` indicating that changes there need to be synchronized over to this file.
1. Check if there's a way to share the settings, e.g. does `grunt-browserify` take options from `package.json` by default?